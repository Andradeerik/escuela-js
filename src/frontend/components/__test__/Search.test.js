import React from 'react';
import { create } from 'react-test-renderer';
import Search from '../Search';

describe('Layout component', () => {
  test('Matches the snapshot', () => {
    const search = create(<Search />);
    expect(search.toJSON()).toMatchSnapshot();
  });
});
