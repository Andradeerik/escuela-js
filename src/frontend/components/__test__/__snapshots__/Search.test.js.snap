// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component Matches the snapshot 1`] = `
<section
  className="main"
>
  <h2
    className="main__title"
  >
    ¿Qué quieres ver hoy?
  </h2>
  <input
    className="input"
    placeholder="Buscar..."
    type="text"
  />
</section>
`;
